Strata.lean

```diff
@@ -27,7 +27,6 @@ import Strata.Languages.Dyn.Examples.Examples
 
 
 /- Code Transforms -/
-import Strata.Transform.Examples
```

Comment:
What does that mean to remove this import? It was not used, but it's going to be compiled, correct?